### PR TITLE
Youtube clone b

### PR DESCRIPTION
**Pull request by RNtaate**
In this project, my partner and I acquired a deeper understanding on how to embed images and videos in an html page and how to use CSS to style.

Milestones:
- We built the HTML structure and used two methods to embed images: CSS Sprites and Img tags in HTML.
- For responsiveness and arrangement of HTML containers, such as div,  we mostly used 'display: flex'
- For embedding the video, we used the iframe tag in HTML, and CSS to assure responsiveness.
- Applied CSS styling on HTML structure with the goal of cloning the original Youtube page. 

To accomplish the above milestones, we had to improve our HTML and CSS skills through The Odin Project tutorials, that can be found here https://www.theodinproject.com/courses/html5-and-css3/lessons/embedding-images-and-video.

### CODE REVIEW

**Comment by trekab** [comment]
# Changes Required!
Hello @RNtaate and @ldelbel, you have done an impressive job on this project so far. I'd like to point a few issues that will make the project :+1:
* You have [HTML validator](https://validator.w3.org/#validate_by_upload) errors to fix.
* I have noticed a heavy use of id's in your CSS. It is preferable to rely on CSS classes and only consider using id's when forced to. Check out this [link](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) for some explanation.

__Kindly work on the suggested changes and submit another code review request__
_Feel free to contact me on slack if necessary_
__Treasure Kabareebe__

**Review by davidmrtz-dev — CHANGES_REQUESTED** [review]
## Hi @ldelbel @RNtaate  . 👨‍🚀  
### Very good job so far.
* Everything is positioned correctly 
* Your README file is clear and descriptive.
* The Html Validator doesn't drop any warning or error.
However, the following issues need to be implemented/fixed to consider this project as approved  👨‍💻  ⬇️ 

****
- [ ] Kindly please edit your pull request to be more descriptive, you can implement a list with a short description of what you did on this project with 4 or 5 points.
Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/) for more information. 
****
- [ ] Kindly please be more descriptive on your commits messages of what you did, like: "Changing id's to classes" instead of "Updating x file"m this is important because helps to the person that is checking your code to find the changes that he's looking.
****
- [ ] As the previous TSE's said, kindly change the id's to classes, this is important because normally the id's are reserved to work with JS. At this moment you're not going to work with Js but it's important to follow this standard.
****

### **Please kindly make the changes required and submit for re-review once you are ready!** 🙏 

****

### Status: Changes required. ♻️  👀 
****

Happy coding! 😄 💻  


**[TSE:  David Elí]** 👨‍💻 
Slack: @ David Eli Martinez Garcia

**Comment by ldelbel** [comment]
Notes:
- The remaining ID in the document was left there to differentiate the first thumbnail from the rest, for styling purposes. We believe it was necessary, because the element already has a class.
- At this milestone, we edited the original Pull Request description.

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,413 @@
+* {
```
Comment (santiago-dev96) [inline styles.css:1]: This selector should be `*, *::before, *::after`. Take a read about it [here](https://www.paulirish.com/2012/box-sizing-border-box-ftw/)

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+}
+
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #f9f9f9;
+  display: block;
```
Comment (santiago-dev96) [inline styles.css:12]: This rule is unnecesessary, the `body` has a `display: block` by default.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+  margin-right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+nav {
+  position: sticky;
+  top: 0;
+  width: 100%;
```
Comment (santiago-dev96) [inline styles.css:31]: By default, elements with a display of `block` (like the `nav` element) take 100% of the width of their parent elements.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+  background-color: white;
+  padding: 13px 0 13px 10px;
```
Comment (santiago-dev96) [inline styles.css:33]: Try to make your code more concise.
```suggestion
  padding: 13px 0;
```

**Review by santiago-dev96 — APPROVED** [review]
# Your project has been approved!

Good work to both of you! I left some suggestions only, I also noticed that the page has no footer, but that's optional for this project (I think the page should have one though). You can go ahead with the next project, congratulations! Have a nice day. :wave:

**Comment by santiago-dev96** [comment]
I almost forget; your __live demo__ link doesn't point in the right direction.

**Pull request merged**